packages/tools/kadena-cli/src/networks/utils/networkHelpers.ts

```diff
@@ -120,8 +120,6 @@ export async function ensureNetworksConfiguration(): Promise<void> {
     mkdirSync(defaultNetworksPath, { recursive: true });
   }
 
-  if (readdirSync(defaultNetworksPath).length !== 0) return;
```

Comment:
Moved this check also as part of network select prompt.

As part of init config also this method is getting called. So, if user (assume he already have its own custom network) wants to create a default networks config later then this check makes it impossible to create it.